### PR TITLE
Add pubstorm deployment instructions to docs

### PR DESCRIPTION
**Pull request by ajhit406**
A lot of Nitrous and PubStorm users are developing and hosting Jekyll projects, we thought it would be helpful to have the instructions here. PTAL, thanks!

### CODE REVIEW

File: site/_docs/deployment-methods.md [hunk 1]
```diff
@@ -225,3 +225,13 @@ Setting up Kickster is very easy, just install the gem and you are good to go. M
...
+
+## PubStorm
+
+[PubStorm](https://www.pubstorm.com) is a free front-end and static-site publishing platform built by [Nitrous](https://www.nitrous.io). PubStorm is distributed as a node package and can be installed by running `npm install -g pubstorm`. You can create a free account by running `storm signup`.
+
+To publish your site, run `storm init` from the root of your project and enter `_site` as the project path when prompted. You can the run `jekyll build` to build your site and then run `storm deploy` to publish your site in seconds.
+
+PubStorm offers a pre-configured CDN, free custom domains, SSL certs, rollbacks, collaboration and more. To configure additional features, [follow the instructions on the PubStorm help site](help.pubstorm.com).
```
Comment (parkr) [inline site/_docs/deployment-methods.md]: This `help.pubstorm.com` needs the protocol or else it'll be a link to our docs 😄

**Comment by ajhit406** [comment]
Good catch, added protocol to http://help.pubstorm.com. PTAL!

**Comment by envygeeks** [comment]
:shipit:

**Comment by parkr** [comment]
Thanks!

@jekyllbot: merge +site

**Pull request merged**